www/model/errors.ts

```diff
@@ -55,78 +55,52 @@ export class FeedbackSubmissionError extends OutlineError {
   }
 }
 
+// Error thrown by "native" code.
+// Must be kept in sync with its Cordova doppelganger:
+//   cordova-plugin-outline/outlinePlugin.js
 export class OutlinePluginError extends OutlineError {
```

Comment:
I found this name rather confusing. What's an "Outline Plugin"? I thought that was something that was added, but then realized it's about the Cordova plugin. Maybe it should be `OutlineCordovaError` or `OutlineCordovaPluginError`

Reply:
I agree it's confusing. I think we should avoid including either "plugin" and "Cordova" in the name because the Windows/Linux clients don't use either. For now I was happy to clarify this class' purpose, having discovered that `windows_connection.ts` had been throwing an `OutlineNativeError` even though it's **logically** an `OutlinePluginError`.

How about a TODO for now? :-)

Reply:
A TODO is fine.
We should consider having the plugin in this repository. Why is it separate?

Reply:
It is in this repository, at `cordova-plugin-outline/`.

I added a TODO to rename this class.